dom.bs

```diff
@@ -5713,9 +5718,11 @@ interface ShadowRoot : DocumentFragment {
   readonly attribute ShadowRootMode mode;
   readonly attribute Element host;
   attribute EventHandler onslotchange;
+  readonly attribute SlotAssignmentMode slotAssignment;
```

Comment:
I think this should be listed under `mode`.

Reply:
moved.